spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20160102
-SPK_REV = 10
+SPK_VERS = 20170422
```

Comment:
I'd change this to 17.4 if you can, as it'll let the users know which version they are using (even if we're using git hashes in the background)